### ISSUE TITLE
Weird custom event issue

### ISSUE THREAD

**Issue by Feerko** [MundoSK/MundoSK#21]
### Bug Report

Skript Version:
2.2-dev31c

MundoSK Version:
newest from github

Minecraft Version:
1.12.2

### Full Code
```vbs
function timer(w: world, n: number=11, prefix: text="") :: boolean:
  set {_sec} to {arena::%{_w}%::time}

  set {_min} to floor({_sec}/60)
  set {_sec} to {_sec}-({_min}*60)

  if {_min} is smaller than 10:
    set {_min} to "0%{_min}%"

  if {_min} is "00":
    call custom event "%uncolored {arena::%{_w}%::round}%" to details {_w}
    call custom event "%uncolored {arena::%{_w}%::round}% %{_sec}% sek" to details {_w}
    call custom event "%uncolored {arena::%{_w}%::round}% %{_sec}% sek %{arena::%{_w}%::typ}%" to details {_w}

  else:
    if {_sec} is smaller than 10:
      set {_sec} to "0%{_sec}%"
      
    call custom event "%uncolored {arena::%{_w}%::round}%" to details {_w}
    call custom event "%uncolored {arena::%{_w}%::round}% %{_min}%:%{_sec}%" to details {_w}
    call custom event "%uncolored {arena::%{_w}%::round}% %{_min}%:%{_sec}% %{arena::%{_w}%::typ}%" to details {_w}

  remove 1 from {arena::%{_w}%::time}
  return true

# in mini game:
set {_w} to world "arena1"
set {arena::%{_w}%::time} to 150
set {arena::%{_w}%::round} to "Refill"
loop {arena::%{_w}%::time}+1 times:
  if timer({_w}, 10) is false:
    stop
  wait 1 second


# event
evt "Refill 02:40":
  set {_w} to event-world
  set {arena::%{_w}%::falldmg} to true
```

### Steps to Reproduce the Issue
it's only parts from code. This code create 3 customs events every second in my mini game. It's works on 1.7.5 but doesn't work on newest version. That's weird because when i'm call simple custom event like "test" it's works. Call custom event "%{_test}%" works too, but there is an error when call custom event "%{test::%event-world%::test}".

Sorry, idk how can i explain it better.

**Comment by Feerko** [comment]
There is no error. Can you enter the discord? I will show you an issue on my server.

**Comment by Feerko** [comment]
@TlatoaniHJ after update issue still exists

**Comment by TlatoaniHJ** [comment]
Which version have you updated to? I will not be able to test for at least a week unfortunately so I can’t do much about the problem right now.

**Comment by Feerko** [comment]
1.8.6-BETA.10 but it's caused for every version after 1.7.5-BETA.10.27

**Comment by TlatoaniHJ** [comment]
I appreciate your concern for fixing bugs in BETA versions, but the point of BETA versions is that they have features that are still in development and so basically they are supposed to have bugs. Anyway, there have been no changes to custom events in BETA versions so I recommend you just use MundoSK 1.8.5 while trying to get this bug fixed.

**Comment by Feerko** [comment]
@TlatoaniHJ 
1.7.5*
https://github.com/MundoSK/MundoSK/issues/14 i think it breaks some events.

**Comment by TlatoaniHJ** [comment]
Issue resolved in MundoSK 1.8.6-BETA.53.
